__docs__/wonder-blocks-link/link.stories.tsx

```diff
@@ -197,9 +198,100 @@ export const OpensInANewTab: StoryComponentType = () => (
 
 OpensInANewTab.parameters = {
     docs: {
-        storyDescription: `When \`target="_blank"\`, the external icon is
-        automatically added to the end of the link. This indicates that the link
-        will open in a new tab.`,
+        storyDescription: `When \`target="_blank"\`, the external icon is automatically
+        added to the end of the link. This indicates that the link will open in a new
+        tab.`,
+    },
+};
+
+export const StartAndEndIcons: StoryComponentType = () => (
+    <View>
+        {/* Default (dark) */}
+        <View style={{padding: Spacing.large_24}}>
+            <Link
+                href="#"
+                startIcon={icons.add}
+                style={styles.standaloneLinkWrapper}
+            >
+                This link has a start icon
+            </Link>
+            <Link
+                href="#"
+                endIcon={icons.search}
+                kind="secondary"
+                style={styles.standaloneLinkWrapper}
+            >
+                This link has an end icon
+            </Link>
+            <Link
+                href="#"
+                startIcon={icons.caretLeft}
+                endIcon={icons.caretRight}
+                style={styles.standaloneLinkWrapper}
+            >
+                This link has a start icon and an end icon
+            </Link>
+            <Link
+                href="#"
+                endIcon={icons.info}
+                target="_blank"
+                style={styles.standaloneLinkWrapper}
+            >
```

Comment:
Do you you think you could add an example that's explicitly multiline with the start and end icons? One approach is to add a max width to the link so that it wraps naturally for the length of the link. You could also have a whole paragraph as a link if you'd prefer.

It's likely that a lot of links won't be designed to wrap with icons anyway, but links can get pretty long when translated to other languages.